lib/web/apiserver.go

```diff
@@ -88,8 +88,8 @@ type Handler struct {
 	// from configuration
 	sshPort string
 
-	// clusterFeatures contain flags for supported and unsupported features.
-	clusterFeatures proto.Features
+	// ClusterFeatures contain flags for supported and unsupported features.
+	ClusterFeatures proto.Features
```

Comment:
Why does this need to be exported?

Reply:
it needed to be exported to be used in the enterprise web handlers to check if cloud was enabled: https://github.com/gravitational/teleport.e/pull/280/files#diff-0c58e7b615d63a03da9676aa4c3add595d3a9e4d63f4d50990d07f6521e2d362R98